src/core/Heading/Heading.tsx

```diff
@@ -37,7 +37,7 @@ const StyledHeading = styled(
     smallScreen,
     className,
     variant,
-    asProp,
+    asProp, // as-property is defined internally as asProp and need to be implemented back if used
```

Comment:
I'd clarify this comment a bit further to explain that asProp needs to be implemented per use case / component if support for that is wanted.